GenderPayGap.WebUI/Views/AccountCreation/CreateUserAccount.cshtml

```diff
@@ -149,29 +156,28 @@
 
                 @(Html.GovUkCheckboxItemFor(m => m.SendUpdates,
                     new LabelViewModel {Text = "I would like to receive information about webinars, events, and new guidance"}))
-
                 @(Html.GovUkCheckboxItemFor(m => m.AllowContact,
                     new LabelViewModel {Text = "I'm happy to be contacted for feedback on this service and take part in gender pay gap surveys"}))
-
-
                 @(Html.GovUkButton(new ButtonViewModel
                 {
-                    Text = "Continue",
+                    Text = "Save and Continue",
                     Classes = "govuk-!-margin-top-9"
                 }))
-
+                @{ var reportingUrl = Url.Action("ManageOrganisationsGet", "ManageOrganisations"); }
+                <a href="@(Model.IsPartOfGovUkReportingJourney ? Global.StartUrl ?? reportingUrl : reportingUrl)" class="govuk-button govuk-!-margin-top-9" role="button">Cancel</a>
```

Comment:
Again, this should be styled as a link, not a button, with the `govuk-link` class